config/locales/en/activerecord.yml

```diff
@@ -447,6 +447,7 @@ en:
         question_options: Options
       legislation/question/translation:
         title: Question
+        description: "Description"
```

Comment:
I think we don't need this line, since it's already defined in `I18n.t("attributes.description")` :thinking:.